modules/oadp-ibm-z-test-support.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-ibm-z-test-support_{context}"]
+= OADP testing and support for target backup locations using IBM Z
+
+IBM Z ("bare-metal") running on OCP 4.11 and OADP 1.1.0 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running IBM Z with OCP 4.11 and OADP 1.1.0 against all non-AWS S3 backup location targets.
```

Comment:
@RichardHoch 
Should 'bare-metal' be hyphenated? 
'running with' as above.
'as well' at the end as above